lib/generators/USAGE

```diff
@@ -0,0 +1,125 @@
+        partial.
+
+        This allows for using Bootstrap in your regular Rails stylesheets. If you
+        wish to customize any of the Bootstrap variables, you can do so via the
+        `client/assets/stylesheets/_pre-bootstrap.scss` partial.
+
+    2. Bootstrap via Webpack Dev Server
+
+        When using the webpack dev server, which does not go through Rails,
+        bootstrap is loaded via the `bootstrap-sass-loader` which uses the
+        `client/bootstrap-sass-config.js` file.
+
+    3. Keeping Custom Bootstrap Configurations Synced
+
+        Because the webpack dev server and Rails each load Bootstrap via a different
+        file (explained in the two sections immediately above), any changes to
+        the way components are loaded in one file must also be made to the other
+        file in order to keep styling consistent between the two. For example,
+        if an import is excluded in _bootstrap-custom.scss, the same import should
+        be excluded in `bootstrap-sass-config.js` so that styling in the Rails
+        server and the webpack dev server will be the same.
+
+    4. Skip Bootstrap Integration
+
+        Bootstrap integration is enabled by default, but can be disabled by passing
+        the --skip-bootstrap flag (alias -b). When you don't need Bootstrap in your
+        existing project, just skip it as needed.
+
+* Linters
+
+    The React on Rails generator can add linters and their recommended accompanying
+    configurations to your project. There are two classes of linters: ruby linters
+    and JavaScript linters.
+
+* JavaScript Linters
+
+    JavaScript linters are enabled by default, but can be disabled by passing the
+    --skip-js-linters flag (alias j), and those that run in Node have been add to
+    `client/package.json` under devDependencies.
+
+* Ruby Linters
+
+    Ruby linters are disabled by default, but can be enabled by passing the
+    `--ruby-linters` flag when generating. These linters have been added to your
+    Gemfile in addition to the the appropriate Rake tasks.
+
+    We really love using all the linters! Give them a try.
+
+* Running the Linters
+
+    To run the linters (runs all linters you have installed, even if you installed
+    both Ruby and Node):
+
+    `rake lint`
+
+    Run this command to see all the linters available
+
+    `rake -T lint`
+
+    Here's the list:
+    ==========================================================
+    rake lint               # Runs all linters
+    rake lint:eslint        # eslint
+    rake lint:js            # JS Linting
+    rake lint:jscs          # jscs
+    rake lint:rubocop[fix]  # Run Rubocop lint in shell
+    rake lint:ruby          # Run ruby-lint as shell
+    rake lint:scss          # See docs for task 'scss_lint'
+    ==========================================================
+
+More Details:
+
+    `https://github.com/shakacode/react_on_rails#generator`
```

Comment:
@yorzi this red symbol here means you don't have a newline at the end of the file. Check [this out](https://robots.thoughtbot.com/no-newline-at-end-of-file) for more info.